test/jdk/java/util/Formatter/BasicDateTime.java

```diff
@@ -1687,6 +1687,14 @@ public static void test() {
 
         // Get the supported ids for GMT-08:00 (Pacific Standard Time)
         String[] ids = TimeZone.getAvailableIDs(-8 * 60 * 60 * 1000);
+
+        final List<String> list =  new ArrayList<String>();
+        Collections.addAll(list, ids);
+        list.remove("America/Dawson");
+        list.remove("America/WhiteHorse");
+        list.remove("Canada/Yukon");
```

Comment:
I'd explicitly mention why these time zones are removed from the test.

Reply:
Thanks, Naoto, I have added the information.